.azure_pipelines/step-dotnet-test-suite.yml

```diff
@@ -0,0 +1,8 @@
+steps:
```

Comment:
Once the coreclr test suite is included in code-coverage/nightly/release pipeline, any specific change required to stop the scheduled dotnet pipeline run?

Reply:
It will automatically glitch since the yaml that drives it will disappear, but we should also manually "disable" it which is a mode in which you have your previous runs but you cant start more unless you "enable"